src/locales/en/common.json

```diff
@@ -0,0 +1,90 @@
+{
+    "cancel": "Cancel",
+    "type": "Type",
+    "continue": "Continue",
+    "next": "Next",
+    "back": "Back",
+    "new": "New",
+    "status": "Status",
+    "start": "Start",
+    "end": "End",
+    "completed": "Completed",
+    "advanved": "Advanved",
```

Comment:
Hey @bradleySuira , both the code and word in English have a typo. We’ll fix these two typos in the common.json file using Crowdin, but I think you need to fix it here and other places.

Reply:
thanks @stephien fixed now in all files